### PR TITLE
Remove debug command & add more usages to commands

### PR DESCRIPTION
**Pull request by powercasgamer**
I've removed the debug command. Since they have no use anyway (couldn't find any code for it). And also added more arguments to the export and import command (--upload, --without-groups, etc.)

### CODE REVIEW

File: common/src/main/java/me/lucko/luckperms/common/command/access/CommandPermission.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
     SYNC("sync", Type.NONE),
     INFO("info", Type.NONE),
     EDITOR("editor", Type.NONE),
-    DEBUG("debug", Type.NONE),
+    //DEBUG("debug", Type.NONE), // Intellij says that this doesn't have any usages. Haven't fully removed it incase it is needed.
```
Comment (lucko) [inline common/src/main/java/me/lucko/luckperms/common/command/access/CommandPermission.java:38]: You can go ahead and remove this

File: common/src/main/java/me/lucko/luckperms/common/command/spec/CommandSpec.java [hunk 2]
```diff
@@ -49,7 +49,7 @@
             arg("type", false),
             arg("filter", false)
     ),
-    DEBUG("/%s debug"),
+    //DEBUG("/%s debug"), // also commented this incase it is needed, if it's not needed it'll fully remove it.
```
Comment (lucko) [inline common/src/main/java/me/lucko/luckperms/common/command/spec/CommandSpec.java:52]: You can go ahead and remove this

**Review by lucko — CHANGES_REQUESTED** [review]
Looks good!

File: common/src/main/java/me/lucko/luckperms/common/command/spec/CommandSpec.java (hunk 2 above)
Comment (powercasgamer) [reply common/src/main/java/me/lucko/luckperms/common/command/spec/CommandSpec.java:52]: Alright, will do once I'm home.

**Comment by powercasgamer** [comment]
done

**Review by lucko — APPROVED** [review]

**Pull request merged**